sql/tsql/TSqlLexer.g4

```diff
@@ -642,6 +642,7 @@ TINYINT:                               'TINYINT';
 TORN_PAGE_DETECTION:                   'TORN_PAGE_DETECTION';
 TRACKING:                              'TRACKING';
 TRANSFORM_NOISE_WORDS:                 'TRANSFORM_NOISE_WORDS';
+TRIM:                                  'TRIM';
```

Comment:
Should it be also added to the [`keyword`](https://github.com/antlr/grammars-v4/blob/master/sql/tsql/TSqlParser.g4#L4211) rule?

Reply:
For most functions in `built_in_functions` names are presented in keywords. For an example: `STRING_AGG` is a keyword, `REPLACE` is also the keyword and so on. As I understand, `TRIM` also should be added  to the keywords list for similarity. Or exists a better solution?

Reply:
Thanks!
Sorry, I was inattentive.